src/main/java/org/sonarlint/intellij/ui/SonarLintRulePanel.kt

```diff
@@ -233,6 +232,7 @@ class SonarLintRulePanel(private val project: Project) : JBLoadingPanel(BorderLa
 
     private fun updateHeader(finding: Finding, ruleDescription: ActiveRuleDetailsDto) {
         ruleNameLabel.text = ruleDescription.name
+        ruleNameLabel.setCopyable(true)
```

Comment:
Not only the rule name but also the severity, finding type, rule key, etc., were copyable (from the header).